bootstrap/bootstrap.go

```diff
@@ -17,17 +17,19 @@ package bootstrap
 
 import (
 	"context"
+	"fmt"
 	"os"
 	"os/signal"
 	"sync"
 	"syscall"
 
-	"github.com/edgexfoundry/go-mod-bootstrap/v3/bootstrap/config"
+	bootstrapConfig "github.com/edgexfoundry/go-mod-bootstrap/v3/bootstrap/config"
```

Comment:
Don't see why need to do this -- there are no other "config" imports in this file.

Reply:
Not needed. Removed.